describe/test_describe.py

```diff
@@ -40,3 +52,46 @@ def test_run_local():
             "violin.html",
         }
     )
+
+
+def test_sanity_local():
+    # Setting environment:
+    describe_func = import_function("function.yaml")
+
+    try:
+        describe_run = describe_func.run(
+            name="task-describe",
+            handler='summarize',
+            inputs={"table": DATA_URL},
+            params={"update_dataset": True, "label_column": "label"},
+            artifact_path=os.path.abspath("./artifacts"),
+            local=True
+        )
+
+    except Exception as exception:
+        print(f"- The test failed - raised the following error:\n- {exception}")
+
+
+@pytest.mark.parametrize("param", PARAM)
```

Comment:
you can parametrize over multiple arguments, meaning you can do:
@pytest.mark.parametrize("n_samples", [1,2,3])
@pytest.mark.parametrize("n_features", [10,20,30])
then you will have a permutation of `n_samples`*`n_features`

Now, you kinda missed the point, all the arguments are stacked inside of a dictionary... Can be much nicer.